tests/ga/test_extension.py

```diff
@@ -136,6 +135,8 @@ def mock_http_put(url, *args, **_):
     def test_cleanup_leaves_installed_extensions(self):
         with self._setup_test_env(mockwiredata.DATA_FILE_MULTIPLE_EXT) as (exthandlers_handler, protocol, no_of_exts):
             exthandlers_handler.run()
+            exthandlers_handler.report_ext_handlers_status()
```

Comment:
I separated the calls to run() and report_ext_handlers_status(). Not all of the unit tests require a call to report_ext_handlers_status(), but it is way too hard to separate those that required it from those that don't. I added a call to report_ext_handlers_status() all across the test code. Since we will be abandoning this code base, I do not think the extra work to figure out which test actually need the call is worth doing it.